src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4276,6 +4276,9 @@ .FORWARDHELPCATEGORY Cmdlet
         else {
             $pagerCommand = 'less'
             $pagerArgs = '-Ps""Page %db?B of %D:.\. Press h for help or q to quit\.$""'
+            if ($null -eq (Get-Command -Name $pagerCommand -Type Application -ErrorAction Ignore)) {
+                $pagerCommand = $null
+            }
```

Comment:
It seems it is a dup of line 4313. We could remove this.

Reply:
Not exactly the same, but I can see some efficiency here.  Making a change.